scripts/mdpow-pow

```diff
@@ -36,6 +36,8 @@ directory
 from __future__ import with_statement
 
 import os
+import matplotlib
+matplotlib.use('agg')  # quick non-interactive plotting
```

Comment:
remove

Reply:
We need these line for the script if we want the plot function can be run with script, otherwise there would be an error `_tkinter.TclError: no display name and no $DISPLAY environment variable`

Reply:
We should not prescribe the backend. See https://matplotlib.org/stable/tutorials/introductory/usage.html#backends for more on backends — on Linux it should actually work automatically and switch to agg (so I am not sure why you get the tkinter error). In any case, it's really the user's responsibility to configure their environment appropriately, e.g. by setting the backend in matplotlibrc or by setting the [MPLBACKEND](https://matplotlib.org/stable/faq/environment_variables_faq.html#envvar-MPLBACKEND) environment variable
```
MPLBACKEND=agg
```
(which is what we should do for our tests).